demisto_sdk/commands/download/downloader.py

```diff
@@ -355,7 +364,7 @@ def build_custom_content_object(file_path: str) -> dict:
             'entity': file_entity,
             'type': file_type,
             'file_ending': file_ending,
-        }
+        } if file_type else {}
```

Comment:
so we now ignore all content entities that are not the ones we know, right?

Reply:
yes